content/knowledge-codemagic/flutter-screenshots-stores.md

```diff
@@ -0,0 +1,402 @@
+        GlobalWidgetsLocalizations.delegate,
+      ],
+      locale: locale,
+      theme: ThemeData(
+        platform: (isAndroid ? TargetPlatform.android : TargetPlatform.iOS),
+      ),
+      home: Column(
+        children: [
+          Container(color: Colors.black, height: 24), // fake, black and empty status bar
+          Expanded(child: child),
+        ],
+      ),
+    ),
+  );
+}
+{{< /highlight >}}
+
+The `getScreenWrapper()` function above returns the final screen to screenshot and here are its arguments:
+- The `child` argument is the screen you want to take a screenshot of.
+- The `locale` argument is the language you want to use for your screenshot.
+- The `isAndroid` argument is important here to get a rendering specific to each OS.
+- The `overrides` argument is useful to mock the logic of your app (database or webservices calls for example).
+- In that example, we use black for the status bar color, which is actually a basic rectangle. But you can change it to whatever you want.
+
+In order to get your fonts working, you’ll need to add the following `flutter_test_config.dart` file in your `test/` directory, with the following content:
+
+{{< highlight bash "style=paraiso-dark">}}
+import 'dart:async';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:golden_toolkit/golden_toolkit.dart';
+
+Future<void> testExecutable(FutureOr<void> Function() testMain) async
+{
+  TestWidgetsFlutterBinding.ensureInitialized();
+  await loadAppFonts();
+  return testMain();
+}
+{{< /highlight >}}
+
+There are specific requirements for the screenshots sizes. Here are the size and densities that you can use for both the Google Play Store and the App Store Connect:
+
+| **Device**                        | **Screen size**                          | **Screen density** |
+| --------------------------------- | ---------------------------------------- | ------------------ |
+| Android smartphone                | `1107 x 1968`                            | `3`                |
+| 7 inches Android tablet           | `1206 x 2144`                            | `2`                |
+| 10 inches Android tablet          | `1449 x 2576`                            | `2`                |
+| iPad pro 2nd gen                  | `2048 x 2732`                            | `2`                |
+| iPad pro 6th gen                  | `2048 x 2732`                            | `2`                |
+| iPhone 8 Plus                     | `1242 x 2208`                            | `3`                |
+| iPhone Xs Max                     | `1242 x 2688`                            | `3`                |
+
+Note that while the sizes for the App Store Connect have to be specifically those sizes, the Google Play Store is more permissive. Also, if you want to display what your app looks like on a tablet, prefer the portrait mode (if it still makes sense for your app, of course), so your users can see more screens on the store without any swipe.
```

Comment:
is it referred to as "the App Store Connect" or just "App Store Connect" ?

Reply:
Removed the "the", though I'm not sure if I have to remove it or not.